app/views/pet_applications/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>Application Number: <%= @pet_application.id %></h1>
+<p><strong>Name:</strong> <%= @pet_application.name %></p>
+<p><strong>Address:</strong> <%= @pet_application.street %>, <%= @pet_application.city %>, <%= @pet_application.state %>, <%= @pet_application.zip %></p>
+<p><strong>Description:</strong> <%= @pet_application.description %></p>
+<p><strong>Status:</strong> <%= @pet_application.status %></p>
+
+<p><strong>Pets Applying For:</strong>
+<ul>
+  <% @pet_application.pets.each do |pet| %>
+    <li><%= link_to pet.name, "/pets/#{pet.id}" %></li>
+  <% end %>
+</ul><br>
+
+<% if @pet_application.status == "In Progress" %>
+  <p><strong>Add a Pet to this Application:</strong></p>
+  <%= form_with url: "/pet_applications/#{@pet_application.id}", method: :get, data: { turbo: false } do |f| %>
+    <%= f.label "Pet Name" %>
+    <%= f.text_field :pet_search %>
+    <%= f.submit "Search" %><br>
+  <% end %><br>
```

Comment:
Great thorough explanation of your code here, you really spoke to it very well and articulated every line with a superb sense of understanding. That is exactly what these projects are supposed to be about. It's definitely nice to get to the end, but it's much better to have a full understanding of what you just did, like you displayed in the video.